### ISSUE TITLE
Installation Problem "[Errno] 5 Input/Output Error" 

### ISSUE THREAD

**Issue by yousuckXD** [feren-OS/feren-OS-Bug-Reporting#45]
I am posting this because I encountered a problem during installation that I couldn't find the solution to anywhere. I don't have much Administrator experience with linux mint, however I do have a good understanding of linux from studying RHEL in colledge (still enrolled). During a live installation on my HP laptop (quad core x64 processor, 6GB of Ram) I encountered an error three times when I got to the create a user page. It gave me an error that said this, 

"The installer encountered an error copying files to the hard disk:

[Errno 5] Input/output error

This particular error is often due to a faulty CD/DVD disk or drive, or a faulty hard disk. It may help to clean the CD/DVD, to burn the CD/DVD at a lower speed, to clean the CD/DVD drive lens (cleaning kits are often available from electronics suppliers), to check whether the hard disk is old and in need of replacement, or to move the system to a cooler environment."

Being pretty confident that my hard disk and Ram was fine, I made sure that I booted the installer not in uefi mode (The first time I did I couldn't even get the installer window to show up) and booted it in live mode and tried installing it 3 different times, each with the usb in a different port (just to rule out a faulty port) and still got the same error every time. 

What worked for me was booting the installation into "Start Feren OS no mode set". However, I also didnt choose LVM (which kinda sucks but not worried about it because of what I use the laptop for) or third party software. I believe it really might have been from the third party software.

**Comment by feren** [comment]
Does this issue also occur when installing Mint 18.3 Cinnamon?

**Comment by yousuckXD** [comment]
To be honest with you I wouldn't know. This is my first experience with
linux mint. I know ubuntu is pretty similar (at least in the command line)
and I had never encountered this problem with any other OS. The weird thing
is that when I was finished installing it would not let me log in with the
right password. I I opened a cli session by using I believe ctrl +shift +
F1 and added a new account and put it in the sudoers group, and after I
rebooted I was able to log into the account that I had just created. I
believe it might be an error with the home folder being encrypted. I'm
sorry I didn't really take the time to get debug information.
